PossibleGroupsForMolecules.lyx

```diff
@@ -8435,7 +8434,8 @@ Si
 \begin_inset Text
 
 \begin_layout Plain Layout
-\begin_inset Formula $\textrm{CH_{2}O}$
+CH
+\begin_inset Formula $_{2}$
 \end_inset
```

Comment:
@sichaox why is CH2O being removed and replaced with CH_2? I don't remember any discussion of this.

Reply:
I see now that you were changing the formula for the molecular formula that you added in an earlier commit which is also part of this pull request, but somehow this wasn't shown when looking at "changes from all commits" after pressing "Files Changed (3)" to see all differences between the pull request's version and the main fork (this must be a bug in GitHub):

![image](https://github.com/HPQC-LABS/ZMC/assets/1163013/a289892a-b9a6-4c5b-9d49-902700129e02)